lib/src/rules/give_deprecation_message.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
```

Comment:
nit: year

Reply:
Curiously, I wrote this code in 2018, but waited to send a PR until the analyzer dep was bumped to 0.35.0, so the git commit's date (as seen above) is from Dec 30, 2018. Should the year still read 2019?

Reply:
I'm not a lawyer, so I don't know whether it's suppose to be the date of file creation of the date that it's first committed into the repo, or whether it actually matters. Your call.

Reply:
Done. 2019.